Sources/MockoloFramework/Operations/Generator.swift

```diff
@@ -36,6 +36,7 @@ public func generate(sourceDirs: [String]?,
                      annotation: String,
                      header: String?,
                      macro: String?,
+                     testableImports: [String],
```

Comment:
Since this is optional, make this an optional type to be consistent with other optional args.